packages/terra-brand-footer/src/BrandFooter.jsx

```diff
@@ -54,21 +54,27 @@ const BrandFooter = ({ links, contentLeft, contentRight, contentBottom, ...custo
   const linkslength = links.length;
   return (
     <footer role="contentinfo" {...customProps} className={BrandFooterClassNames}>
-      {linkslength > 0 ? (
+      {linkslength > 0 && (
         <nav className={cx('nav')}>
           <ul className={cx('menu')}>
-            {links.map(link => (
-              <li className={cx('list-item')} key={link.text + link.href}>
-                {link.target !== undefined ? (
-                  <a className={cx('link')} href={link.href} target={link.target} >{link.text}</a>
-                ) : (
-                  <a className={cx('link')} href={link.href} >{link.text}</a>
-                )}
-              </li>
-            ))}
+            {links.map((link) => {
```

Comment:
There is a lot of conditional logic happening here. I am wondering if it would be more readable if this was broken out to create the list items.